npc/re/merchants/hd_refiner.txt

```diff
@@ -105,6 +105,7 @@
 		mes "Well, even if it fails, it only decreases by 1 refine level.";
 		mes "Would you like to continue refining?";
 		next;
+		.@menu$ = "";
 		if (countitem(6635) < .@blacksmith_blessing_count)
 			setarray .@menu$[1], "Yes", "Not yet";
```

Comment:
```suggestion
			setarray .@menu$[0], "", "Yes", "Not yet";
```
Wouldnt this be more readable? Or do I understand it wrong?

Reply:
No you're right it's more readable for me too